pwndbg/commands/heap.py

```diff
@@ -574,11 +603,13 @@ def vis_heap_chunks(address=None, count=None, naive=None):
 
     print(out)
 
+
 def bin_ascii(bs):
     from string import printable
-    valid_chars = list(map(ord, set(printable) - set('\t\r\n')))
+    valid_chars = list(map(ord, set(printable) - set('\t\r\n\x0c')))
```

Comment:
Why?

Reply:
Without this change, in xterm I see the following effect when vis_heap_chunks() attempts to hexdump the form feed (`0x0c`) character:
![Capture](https://user-images.githubusercontent.com/16000770/80233764-dc7a8280-860b-11ea-8d94-af5d37458352.PNG)